src/docker-osx-dev

```diff
@@ -375,17 +381,74 @@ function init_boot2docker {
     log_info "Starting Boot2Docker VM"
     boot2docker start --vbox-share=disable
   fi
+
+  configure_boot2docker
+}
+
+################################################################################
+# docker-machine manipulation
+################################################################################
+
+#
+# Read a json string from stdin and return the value of the key provided as first argument
```

Comment:
Can you add an example call to this documentation? It makes it more obvious what the method does.